conan/tools/cmake/cmake.py

```diff
@@ -57,38 +58,48 @@ def __init__(self, conanfile):
 
         self._generator = configure_preset["generator"]
         self._toolchain_file = configure_preset.get("toolchainFile")
-        self._cache_variables = configure_preset["cacheVariables"]
-
+        self._presets_cache_variables = configure_preset["cacheVariables"]
         self._cmake_program = "cmake"  # Path to CMake should be handled by environment
 
+    @property
+    def _cache_variables(self):
+        """
+        Get all the variables that should be passed the CLI as ``cmake -Dxxxxx -Dyyyyy ....``
+        """
+        cache_variables = {}
+        generator_folder = self._conanfile.generators_folder
+        if self._toolchain_file:
+            if os.path.isabs(self._toolchain_file):
+                toolchain_file = self._toolchain_file
+            else:
+                toolchain_file = os.path.join(generator_folder, self._toolchain_file)
+            cache_variables["CMAKE_TOOLCHAIN_FILE"] = toolchain_file.replace("\\", "/")
+
+        if self._conanfile.package_folder:
+            pkg_folder = self._conanfile.package_folder.replace("\\", "/")
+            cache_variables["CMAKE_INSTALL_PREFIX"] = pkg_folder
```

Comment:
This is the only variable I don't know what to do with it for the "initial cache", but I would take into account that the "initial cache" idea is to simplify the user calling `cmake` manually, so the INSTALL_PREFIX should be anyway specified because it is a local folder where you are going to install the contents.